examples/webpack-fix-style-only-entries/index.js

```diff
@@ -0,0 +1 @@
+console.log('ok');
```

Comment:
This just for testing the output?

Reply:
Yes, in this case it's a dummy but some test cases (the end-to-end ones) pick up on the `ok` output. Either way, it's just example code.